eventually/authentication/models.py

```diff
@@ -0,0 +1,145 @@
+"""
+This file contains models for the postgresql database.
+"""
+
+#pylint: disable=W0223
+#The pylint comment above is for ignoring the error below
+#Method 'get_full_name' is abstract in class 'AbstractBaseUser'/'get_short_name'
+#but is not overridden (abstract-method)
+
+from datetime import datetime
+from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.utils.dateformat import format
```

Comment:
for what this import is needed for

Reply:
To show date as a timestamp number.
Line 102 in models.py: 'created_at': format(self.created_at, u'U

Reply:
Redefining built-in 'format' (redefined-builtin)